app/workers/fetch_graph_worker.rb

```diff
@@ -5,11 +5,11 @@ class FetchGraphWorker
   include Sidekiq::Worker
   sidekiq_options retry: 11 # Around 2.5 days of retries
 
-  def perform(pid, user_key)
+  def perform(pid, _user_key)
     # Fetch Work and SolrDoc
     work = ActiveFedora::Base.find(pid)
     solr_doc = SolrDocument.find(pid)
-    user = User.where(email: user_key).first
+    # user = User.where(email: user_key).first
```

Comment:
I think we can remove the commented out line if it's not needed

Reply:
It is. These commented pieces of code will need to be reset. Its on my agenda for #986

Reply:
ok sounds good